src/_components/RoutesComponent.jsx

```diff
@@ -24,12 +27,29 @@ import AddAssetSpec from './AssetSpecs/AddAssetSpecContainer';
 class RoutesComponent extends Component {
   checkAuthentication = () => !!(localStorage.getItem('art-prod-web-token'));
 
+  checkAdmin = () => {
+    const token = localStorage.getItem('art-prod-web-token');
+
+    if (token) {
```

Comment:
Always `return early` where possible. This is one of the principles of clean code. Therefore, `checkAdmin` can be refactored to:
```js
checkAdmin = () => {
  const token = localStorage.getItem('art-prod-web-token');
  if (!token) {
    return false;
  }
  // The logic for getting the admin value goes here
}